ArcGISRuntimeSDKQt_QMLSamples/Scenes/AnimateImagesWithImageOverlay/AnimateImagesWithImageOverlay.qml

```diff
@@ -28,7 +28,11 @@ Rectangle {
 
     readonly property var timerIntervals: [17,34,68]
     readonly property var imageFrameRefreshRate: ["Fast","Medium","Slow"]
-    readonly property url dataPath: System.userHomePath +  "/ArcGIS/Runtime/Data/3D/ImageOverlay/PacificSouthWest"
+    readonly property url dataPath: {
+        Qt.platform.os === "ios" ?
+                    System.writableLocationUrl(System.StandardPathsDocumentsLocation) + "/ArcGIS/Runtime/Data/3D/ImageOverlay/PacificSouthWest" :
+                    System.writableLocationUrl(System.StandardPathsHomeLocation) + "/ArcGIS/Runtime/Data/3D/ImageOverlay/PacificSouthWest"
```

Comment:
I don't think the qml ternaries are on the same level of boilerplate as C++, so I would not argue they need a common helper.